.github/ISSUE_TEMPLATE/submission_request.md

```diff
@@ -6,28 +6,27 @@ labels: ''
 assignees: ''
 ---
 
-**Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further.
+<!-- **Note**: Please read [this article](https://docs.genicsblog.com/author/create-a-submission-request) before continuing further. -->
 
 ### Idea
-
-In this space, explain what your article is about.
+<!-- In this space, explain what your article is about. -->
 
 ### Topics
 
-Checkout the [existing topics](https://genicsblog.com/topics).
+<!-- Checkout the [existing topics](https://genicsblog.com/topics). -->
 
-**Category**: The category that your article belongs to (maximum 1).
+**Category**: <!-- The category that your article belongs to (maximum 1). -->
 
-**Tags**: The tags that your article should have (maximum 3).
+**Tags**: <!-- The tags that your article should have (maximum 3). -->
 
 ### Cross-posting?
 
-Is your article already published somewhere else? If yes, please provide the link to the original article.
+<!-- Is your article already published somewhere else? If yes, please provide the link to the original article.
 
-You must be the original author of that article to re-post at Genics Blog.
+You must be the original author of that article to re-post at Genics Blog. -->
 
 ### Remarks
 
-Please add anything you would like to share.
+<!-- Please add anything you would like to share. -->
 
-**Note:** Please remove all the unnecessary lines and comments before submitting the issue.
+<!-- **Note:** Please remove all the unnecessary lines and comments before submitting the issue. -->
```

Comment:
This too.

Reply:
I didn't get it here, there is no link

Reply:
I mean the `**Note**` markdown could be removed 😄

Reply:
Oh ok

Reply:
Done